websites/R/Rust Lang Book/metadata.json

```
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "534614025271771158",
+		"name": "eli_yya#3893"
```

Discriminator added mistakenly by outdated `pmd` program.
```suggestion
		"name": "eli_yya"
```